pkg/liqo-controller-manager/foreign-cluster-operator/resourceRequest.go

```diff
@@ -98,3 +98,7 @@ func resourceRequestLabels(remoteClusterID string) map[string]string {
 		consts.ReplicationDestinationLabel: remoteClusterID,
 	}
 }
+
+func getResourceRequestNameFor(cluster discoveryv1alpha1.ClusterIdentity) string {
+	return fmt.Sprintf("resourcerequest-%s", cluster.ClusterName)
+}
```

Comment:
I feel it is probably useless to prepend `resourcerequest`, given that is implied by the resource type. Should the name be based on the unique name version?

Reply:
Resource requests are created in the tenant namespace, so I believe in principle they might as well be named `request`. I'll need to check that.